script/prepare-release.js

```diff
@@ -79,7 +79,9 @@ async function getReleaseNotes (currentBranch) {
     head: currentBranch
   }
   let releaseNotes = ''
-  if (!args.automaticRelease) {
+  if (args.automaticRelease) {
+    releaseNotes = '## Bug Fixes \n\n'
```

Comment:
Not necessarily true though.

Maybe '## Automatic Release \n\n' ?

Reply:
@ckerr given that it is a header `Automatic Release` doesn't make much sense.  How about Bug Fixes/Changes?

Reply:
d'oh of course. Yes something like that would be fine.